src/lib/datepicker/month-view.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  Component,
+  ViewEncapsulation,
+  ChangeDetectionStrategy,
+  Input,
+  EventEmitter,
+  Output,
+  AfterContentInit
+} from '@angular/core';
+import {MdCalendarCell} from './calendar-table';
+import {CalendarLocale} from '../core/datetime/calendar-locale';
+import {SimpleDate} from '../core/datetime/simple-date';
+
+
+const DAYS_PER_WEEK = 7;
+
+
+/**
+ * An internal component used to display a single month in the datepicker.
+ * @docs-private
+ */
+@Component({
+  moduleId: module.id,
+  selector: 'md-month-view',
+  templateUrl: 'month-view.html',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MdMonthView implements AfterContentInit {
+  /**
+   * The date to display in this month view (everything other than the month and year is ignored).
+   */
+  @Input()
+  get date() { return this._date; }
+  set date(value) {
+    this._date = this._locale.parseDate(value) || SimpleDate.today();
+    this._init();
+  }
+  private _date = SimpleDate.today();
```

Comment:
When it's acting as backing for a public property with a getter/setter isn't it nicer to have it next to its getter/setter?